tornado/websocket.py

```diff
@@ -238,6 +238,16 @@ def get_compression_options(self):
         """
         return None
 
+    def get_handshake_response_headers(self):
```

Comment:
Oh, I see now how it's used. Still, it might be better to just use `self._headers` and not introduce the new interface.